frontend/src/app/core/global_search/input/global-search.component.sass

```diff
@@ -45,33 +45,39 @@
     margin-left: -5px
 
   &--wp-content
-    display: grid
-    grid-template-columns: 50% 1fr auto
-    grid-template-areas: "project idlink status"
+    display: flex
+    flex-direction: column
     font-size: 0.8rem
-    padding: 5px 0 5px 0px
+    padding: 5px 0
+
+  &--wp-line
+    display: flex
+    flex-wrap: nowrap
+    gap: var(--base-size-8)
+    font-size: 13px
```

Comment:
Why 13px? We don't have that anywhere else. Please use a Primer font variable to map it to 12px